addons/ofxSvg/src/ofxSvg.h

```diff
@@ -17,15 +17,17 @@
 
 class ofxSvg {
 public:
+	ofxSvg();
+	ofxSvg(const of::filesystem::path & fileName);
 	~ofxSvg();
-
+	
```

Comment:
I would actually get rid of the ofxSvg() and ~ofxSvg() and let them just be default implementations. 

I was going to suggest using the default keyword but I think you don't need any of the other destructors / constructors if you are just defining:
`	ofxSvg(const of::filesystem::path & fileName);`

Reply:
I think ~ofxSvg() is being used, there is a paths.clear() there

Reply:
and I'm wondering if we need ofxSvg() to use it in a traditional way lets say
```
ofxSvg svg;
```

Reply:
> and I'm wondering if we need ofxSvg() to use it in a traditional way lets say
> ofxSvg svg;

ahh yeah, in that case default constructor, default destructor and default copy constructor might be good too. 
https://en.cppreference.com/w/cpp/language/default_constructor 

> I think ~ofxSvg() is being used, there is a paths.clear() there

that should just happen automatically when the class is destroyed - so I think paths.clear and ~ofxSvg()  implementation is not needed.

Reply:
good! I'll test here

Reply:
Nice! 
destruct the destructor ok
and tidy up file loading. now it correctly checks if file is not found.